config/vufind/Aleph.ini

```diff
@@ -95,3 +95,7 @@ extraHoldFields = comments:requiredByDate:pickUpLocation
 ; \VuFind\Cache\Manager cache you would like to use for storing data ("object" is recommended).
 [Cache]
 ;type = object
+
+[TransactionHistory]
```

Comment:
Is there a reason we're configuring this independently in every ILS driver, rather than making history a global config.ini setting? Perhaps this makes sense due to some MultiBackend-related scenario; I'm just double-checking whether or not it makes sense to centralize the configuration.

Reply:
For us it's the delicate nature of storing the loan history in the first place, and if the library in question hasn't got the required policy etc. in place, it's better to leave it completely disabled. Therefore we really need the granularity.

Reply:
Okay, makes sense to me!